example/v1alpha1/main.go

```diff
@@ -41,7 +41,8 @@ func main() {
 	}
 
 	oPID := "scanning-customer"
-	o := Occurrence(createdN.Name)
+	o := Occurrence(createdN.Name, fmt.Sprintf("projects/%v/occurrences/%v", nPID, oPID))
```

Comment:
The occurrence does not exist in the note project namespace - oID is correct here.